README.md

```diff
@@ -115,6 +115,10 @@ CNCF Projects may freely select their own tools, produce their own documentation
 - [Scarf](https://scarf.sh) for advanced analytics for container & artifact distribution, package installation, and web traffic to source documentation
 - [Peritus.ai](https://peritus.ai/) for Machine Learning analytics and self-service for developer communities
 - [Curiefense](https://curiefense.io) for application layer protection (web/API) - WAF, DDoS, Rate limiting, and more
+- [CLOMonitor](https://clomonitor.io) scans repositories daily, checking for adherence to a range of best practises providing a way for end users to assess the health of open source projects from multiple points of view
```

Comment:
Is our standard `en-us` or `en-ie` for the word "practices"? 😊

Reply:
I think we usually default to `en-us`

Reply:
(that said, [our style guide](https://github.com/cncf/foundation/blob/main/style-guide.md) is pretty sparse)

Reply:
I love that that style guide definitively addresses the [long-standing open source and cloud native hyphen tiff we've had with Grammarly](https://twitter.com/DanielKrook/status/1347606838632734723)! 

On the practices thing... we have already [set the precedent here in the README](https://github.com/cncf/servicedesk#program-management) so let's go with west-pond style.

Reply:
Hiberno English is the best English, the plurality of practises defendable here, I think, by the wide range of actions required (and checked) to make a project truly great 😜 

Happy to en-us it mind. Nice catch!